git_theta/updates/sparse.py

```diff
@@ -17,7 +19,18 @@ def name(self):
     async def calculate_update(
         self, parameter: Parameter, previous_parameter: Parameter
     ) -> Parameter:
-        return parameter - previous_parameter
+        diff = parameter - previous_parameter
+        update = scipy.sparse.csr_matrix(np.reshape(diff, (-1,)))
```

Comment:
Does the dense -> csr transform do any sort of thesholding or is it only looking for actual 0's?

We should add a comment about what it does and think about if we want to add some sort of threshold for what we consider 0 (i.e. < 1e-12 could be converted to 0 to create a more sparse tensor).

Reply:
>We should add a comment here explaining that.

Added comment on why we need to give the shape for csr_matrix when reconstructing from data. 

>Does the dense -> csr transform do any sort of thesholding or is it only looking for actual 0's?

From what I see in documentation, it looks for actual 0's. There is no parameter to pass in a threshold value. If we plan to have a threshold, we can implement it on the update tensor before passing it to csr_matrix.  I will add a comment in the code on this.